lib/node_modules/@stdlib/random/base/mt19937/lib/factory.js

```diff
@@ -62,19 +62,32 @@ var MATRIX_A = 0x9908b0df;
 * // returns 822569775
 */
 function factory( seed ) {
+	var seedIsArray = false;
 	var state;
+	var mag01 = [ 0x0, MATRIX_A ];
 	var mti;
 	var mt = new Array( N );
```

Comment:
Declare `mt` but do not assign here. Instead, more initialization to L95 or thereabout. Why? Otherwise, if provided a bad input, we'll have unnecessarily allocated memory, just making the situation worse.